lib/location/method_gnss.c

```diff
@@ -57,7 +57,7 @@ BUILD_ASSERT(
 #define AT_MDM_SLEEP_NOTIF_START "AT%%XMODEMSLEEP=1,%d,%d"
 #endif
 #if (defined(CONFIG_NRF_CLOUD_AGNSS) || defined(CONFIG_NRF_CLOUD_PGPS))
-#define AGNSS_REQUEST_RECV_BUF_SIZE 3500
+#define AGNSS_REQUEST_RECV_BUF_SIZE 3600
```

Comment:
I was wondering, is it possible to define this value in a Kconfig or a header? Because it is copied in multiple places.

Reply:
That's a good question. I think a define in a header would be better then a Kconfig option, because the maximum size does not change, unless a new satellite constellation or other assistance data is added. This is nRF Cloud specific, so the most logical place would probably be an nRF Cloud library header, probably `nrf_cloud_agnss.h` or `nrf_cloud_rest.h`. The buffer is only used with REST and CoAP, and CoAP uses structures defined in `nrf_cloud_rest.h`. But still, I'm leaning towards `nrf_cloud_agnss.h` with a comment, that it's only for REST and CoAP.